netlify.toml

```diff
@@ -7,6 +7,7 @@
 
 [build]
   base = "docs/"
+  command = "yarn;yarn build-storybook;"
```

Comment:
@maestromac, I was wondering if we should run `yarn test` before building storybook. Storybook won't build if the components are borked, so maybe running `yarn test` here is overkill.

Reply:
yeah, i agree. There's no need to run `yarn test` there.